src/message.coffee

```diff
@@ -21,15 +21,15 @@ class Message extends EventEmitter
 
     # Keep track of when this message actually times out.
     @timedOut = false
-    do trackTimeout = =>
+    do trackTimeoutFunction = =>
```

Comment:
We try to avoid including types in variable names, so I would revert this rename and assign the return value of `setTimeout` to something called `trackTimeoutId` or similar.